Sources/MapboxCoreNavigation/EHorizon/EHorizonOptions.swift

```diff
@@ -0,0 +1,38 @@
+import Foundation
+import MapboxNavigationNative
+
+public struct EHorizonOptions {
+
+    /** The minimum length of the EHorizon ahead of the current position. */
+    public let length: Double
+
+    /** The expansion strategy to be used. */
+    public let expansion: UInt
```

Comment:
What strategies are available? Could we introduce an enum or is it just a boolean flag?

Reply:
Yeah, it should be enum I beleive: 0, 1, 2 is available.

Reply:
https://github.com/mapbox/mapbox-navigation-native/pull/2496/files#diff-18df9a7b99e917622e94af64345e38aa0c777cef30aabccaac494d24f79a6babR132

Reply:
I’m leaving this property as a number because it isn’t technically a closed set of values.